publications/joss/paper.md

```diff
@@ -207,7 +197,7 @@ The following open-source tools are available to either perform forward modeling
 (based on Bayesian statistics and posterior distribution):
 [`Starfish`](https://starfish.readthedocs.io/en/latest/index.html) [@Czekala2015starfish], 
 [`petitRADTRANS`](https://gitlab.com/mauricemolli/petitRADTRANS) [@Molliere2019],
-[`petitRADTRANS`](https://gitlab.com/mauricemolli/petitRADTRANS) [@MacDonald2023],
+[`POSEIDON`](https://github.com/MartianColonist/POSEIDON) [@MacDonald2023],
```

Comment:
I changed the code base and URL for this software based on the citation `@MacDonald2023`, but please confirm this change is correct.